tests/basic/storage_pool_test.go

```diff
@@ -956,6 +969,32 @@ func waitForPoolToBeResized(expectedSize uint64, poolIDToResize string, isJourna
 	return err
 }
 
+func getPoolStatus(poolID string) (*api.StoragePoolOperation, error) {
```

Comment:
I would rename this function to `getPoolLastOperation()` since that is what its returning

Reply:
fixed